packages/browser-lib/src/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export { base, recommended, full } from '@sa11y/preset-rules';
+export { sortViolations } from '@sa11y/format';
+export { assertAccessible } from '@sa11y/assert';
+
+export const nameSpace = 'sa11y';
```

Comment:
This is usually "namespace" (all lowercase).

Reply:
Fixed.